public/static/docs/api-reference/get_url.md

```diff
@@ -0,0 +1,85 @@
+# dvc.api.get_url()
+
+Return the URL to the storage location of the target data (`repo`/`path`).
+
+## Signature
+
+```py
+get_url(path, repo=None, rev=None, remote=None)
+```
+
+## Description
+
+Returns the full URL string (`str` type) to the physical location (in a
+[DVC remote](/doc/command-reference/remote)) where a target <abbr>data
+artifact</abbr> specified by its `path` in a `repo` (<abbr>DVC project</abbr>)
+is stored.
+
+⚠️ Note that the returned URL is formed by analyzing the corresponding
+[DVC-file](/doc/user-guide/dvc-file-format) (see [Examples](#examples) below).
+**There is no guarantee that the file actually exists in that location**. Please
+keep this in mind when using the URL string in your code.
+
+💡 Having the resource's URL, it should be possible to download it directly with
```

Comment:
it can be a variety of different URLs ... and a variety of tools. For example, why not `aws s3 cp` vs `boto3`, I think , we need to simplify and generalize, too keep only the essence of the paragraph

Reply:
The essence of this paragraph is to explain the use case(s) for this function. Maybe I just don't get what its purpose is? Why would you ever need the artifact's storage URL if not to go fetch it yourself?

The phrase "tool such as" implies it can be a variety of tools, but I was just focusing on Python packages here, since it's in the Python API ref. Not sure it makes sense to mention CLI tools here, as you're already in a Python source code context.

To simply generalize this, I would just remove the tool examples, would that be better?

Reply:
yes, my point was about your examples of tools are way too niche

Reply:
I see. I'm assuming HTTP and S3 are some of the more widely used types of DVC remotes but honestly I don't know. I'll check with Ruslan (I assume he knows).
- For local files there's always the `open()`builtin (bu using it in an API ref example may be confusing since we also have `dvc.api.open()`)
- For SSH, there's [paramiko](https://github.com/paramiko/paramiko
- For HDFS there's [mtth/hdfs](https://github.com/mtth/hdfs) and [snakebite](https://github.com/spotify/snakebite)
- All the other types have official or well known libraries (`azure-storage-blob`, `google-api-python-client `,  `google-cloud-storage`, `aliyun-oss-python-sdk`)

I guess we don't want to mention all of them, but not sure how to simplify either. Do you mean to just remove all references? I could also put them in an expandable section/note.

Reply:
I mean even a list like `curl`, `aws s3`, `boto3` should be easier to understand. Some of the tools you mention I hear first time about, means probably they are too niche. No need to list a lot here

Reply:
Also, not sure we need an emoji here.

Reply:
> I mean even a list like curl, aws s3, boto3 should be easier to understand.

That's what this is trying to accomplish, yes. It looks long in the code but the rendered text is short (just links).

I've changed it to the most common ones (S3, SSH) now, after asking on #engineering channel:

![image](https://user-images.githubusercontent.com/1477535/75409815-5bd72880-58df-11ea-9b88-ab38e65b3147.png)

> not sure we need an emoji here

I think it's appropriate because we use 💡 for 3rd party tools/code tips & hints (see #855). But no strong opinion, I'll remove it if you prefer. Please lmk

Reply:
p.s. resolving this one for now.

Reply:
it can be downloaded with simple tools like curl or wget as well, not even libraries only